Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpAnimationImporter.cpp

```diff
@@ -457,7 +457,10 @@ namespace AZ
                     {
                         aiBone* bone = mesh->mBones[boneIndex];
 
-                        boneList.insert(bone->mName.C_Str());
+                        if (!IsPivotNode(bone->mName))
```

Comment:
nit: any danger the bone has a null ptr for a name?

Reply:
I've added some null checks + error messages